### PR TITLE
Undo fortran cases

### PR DESCRIPTION
**Pull request by znicholls**
# Pull request

Please confirm that this pull request has done the following:

- [x] Tests added
- [x] Documentation added (where applicable) (N/A)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable) (N/A)
- [x] Description in ``CHANGELOG.rst`` added

## Adding to CHANGELOG.rst

Please add a single line in the changelog notes similar to one of the following:

```
- (`#XX <http://link-to-pr.com>`_) Added feature which does something
- (`#XX <http://link-to-pr.com>`_) Fixed bug identified in (`#XX <http://link-to-issue.com>`_)
```

### CODE REVIEW

File: pymagicc/io.py [hunk 1]
```diff
@@ -2679,7 +2687,7 @@ def _write_variable_datablock_radforcing(self, output, data_block, units_level):
             data_block, units_level
         )
 
-        units = [u.replace("Wpermsuper2", "W/m2") for u in units]
+        units = [u.replace("W_per_msuper2", "W/m2") for u in units]
```
Comment (lewisjared) [inline pymagicc/io.py:2690]: Why `super` not `_super_` for consistency?

Comment (znicholls) [reply pymagicc/io.py:2690]: I'm using "_" for space at the moment and I don't think pint would work if we had "W m ^ 2"

Comment (znicholls) [reply pymagicc/io.py:2690]: although maybe it would...

Comment (lewisjared) [reply pymagicc/io.py:2690]: Ahh makes sense. It thought it was to tokenise it

**Pull request merged**